agentops/client.py

```diff
@@ -75,7 +76,7 @@ def __init__(
         tags: Optional[List[str]] = None,
         override: Optional[bool] = None,  # Deprecated
         instrument_llm_calls=True,
-        auto_start_session=True,
+        auto_start_session=False,
```

Comment:
Why switch this?

Reply:
It's still True when you init()
init(auto_start_session=True)
Client(auto_start_session=False)

Just not if you don't call init()

Changing auto_start_session default to True only if init() was explicitly called. Otherwise had bug where initializing Client singleton with record_function() would auto_start_session when it was just being decorated. Meaning partner frameworks where agentops was integrated but optional (e.g. multion) would try to start_session() even if user didn't explicitly use agentops.

Reply:
we had a discussion in the office about this change